versions/v1/template/server.ts

```diff
@@ -1,91 +1,115 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-nocheck
 // This file should be placed alongside the user's files.
 
-import {makeExecutableSchema, mergeSchemas} from '@graphql-tools/schema';
-import {schemaFromExecutor, wrapSchema} from '@graphql-tools/wrap';
-import {buildHTTPExecutor} from '@graphql-tools/executor-http'
-import {ApolloServer} from 'apollo-server-express';
-import {deepCloneVMFunction} from "./deep-clone-vm";
-import {CreateServerConfig, GraphQLServer, HelpersMap, ResolversMap} from "./types";
-import {createSatsumaKnex} from "./knex";
+import { buildHTTPExecutor } from "@graphql-tools/executor-http";
+import { makeExecutableSchema, mergeSchemas } from "@graphql-tools/schema";
+import { schemaFromExecutor, wrapSchema } from "@graphql-tools/wrap";
+import { ApolloServer } from "apollo-server-express";
 
+import { deepCloneVMFunction } from "./deep-clone-vm";
+import { createSatsumaKnex } from "./knex";
+import {
+  CreateServerConfig,
+  GraphQLServer,
+  HelpersMap,
+  ResolversMap,
+} from "./types";
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 let resolvers = {};
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 let typeDefs = "";
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 let helpers = {};
```

Comment:
Why are these set later but not used? Is this right?

Reply:
They're used as default params.

https://github.com/satsuma-xyz/satsuma-querier-codegen/pull/4/files#diff-1665f4aa86c7f2c5bb63c31ea409c9167edcdea31c14db5cdbd3dfcdf1fa4e83R81-R82

https://github.com/satsuma-xyz/satsuma-querier-codegen/pull/4/files#diff-1665f4aa86c7f2c5bb63c31ea409c9167edcdea31c14db5cdbd3dfcdf1fa4e83R103-R104

Reply:
Cool, i'll leave the lint disabling here and let you remove in your PR.